components/scream/src/physics/shoc/atmosphere_macrophysics.hpp

```diff
@@ -214,12 +247,15 @@ class SHOCMacrophysics : public scream::AtmosphereProcess
     view_2d              cloud_frac;
 
     // Assigning local variables
-    void set_variables(const int ncol_, const int nlev_, const int num_qtracers_, const Real z_surf_,
+    void set_variables(const int ncol_, const int nlev_, const int num_qtracers_,
+                       const view_1d_int_const& convert_wet_dry_idx_d_,
+                        const Real z_surf_,
```

Comment:
has an extra white-space.  Not a big deal though, just mentioning.

Reply:
same here

Reply:
@singhbalwinder , still not a big deal, but as long as you may be making other changes.